go-fuzz-build/main.go

```diff
@@ -665,6 +665,20 @@ func (c *Context) clonePackage(p *packages.Package) {
 		c.copyFile(f, dst)
 	}
 
+	// copy subdirs which aren't packages
+	dir := filepath.Dir(p.GoFiles[0])
+	subdirs, err := ioutil.ReadDir(dir)
+	if err != nil {
+		c.failf("failed to scan dir '%v': %v", dir, err)
+	}
+	for _, d := range subdirs {
```

Comment:
Is there a reason not to do a recursive copy of dir, instead of copying all Go files, all non-Go files, and then all subdirs?

Reply:
The reasons are:
1. Keep the existing approach and code style;
2. Total recursive copy of dir will copy subpackage dirs also, what is undesirable, because subpackage may be unnecessary (no dependency), or may be copied twice during process the direct import of it.